components/gnome/mousetweaks/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2015, 2021, Oracle and/or its affiliates.
+#
+
+BUILD_BITS =            64
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mousetweaks
```

Comment:
... and finally, this project is [deprecated](https://gitlab.gnome.org/Archive/mousetweaks), so it cannot be needed for GNOME 3.  The homepage (referenced from `COMPONENT_PROJECT_URL`) says so too.

Reply:
It is Archived, that does not mean it's deprecated. It is a simple piece of software. It looks more finished, which is ok.

Reply:
> It is Archived, that does not mean it's deprecated. It is a simple piece of software. It looks more finished, which is ok.

Yes, Archived could also mean it is not active (see https://gitlab.gnome.org/Archive), but in this case it is deprecated.

What GNOME package depends on mousetweaks?